cc-visionary/Inventory-App
# Fix Edit Product Input Placeholder

How to test:
1. Run the command `npm run start` on the root directory
2. Login as `admin123` for username and `password` for password
3. Click `Inventory` button in the Navbar
4. Try editing a product
5. Remove the texts in the inputs to see the input placeholders
